Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,35 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4970157)

Reply:
Fixed in [2db6c28](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/161/commits/2db6c282219933cc661146c542a210b0f73f44cf)